### DISCUSSION TITLE
Use consistent suffix for cycled UFSDA netcdf files

### DISCUSSION THREAD

**Discussion by RussTreadon-NOAA** [NOAA-EMC/global-workflow#2078] [General]
The current implementation of UFSDA DA uses two suffixes for netcdf files:  `.nc` and `.nc4`.  It seems preferable for the sake of consistency to use a single suffix.    

Discussion among a subset of DAD UFSDA DA marine and atm developers is leaning towards `.nc`.  Both GDASApp and g-w reference cycled UFSDA netcdf files.  As such, this discussion is opened to get the g-w perspective.

**Comment by RussTreadon-NOAA** [comment]
Add @CoryMartin-NOAA , @guillaumevernieres , @ADCollard , and @emilyhcliu for awareness.

**Reply by aerorahul** [reply]
https://docs.unidata.ucar.edu/netcdf-c/current/faq.html#:~:text=NetCDF%20files%20should%20have%20the,nc%22.

**Comment by RussTreadon-NOAA** [comment]
Thanks @aerorahul 

> **What convention should be used for the names of netCDF files?**
NetCDF files should have the file name extension ".nc". The recommended extension for netCDF files was changed from ".cdf" to ".nc" in 1994 in order to avoid a clash with the NASA CDF file extension, and now it also avoids confusion with "Channel Definition Format" files.

`.nc` is the netcdf team recommendation.

**Comment by aerorahul** [comment]
@RussTreadon-NOAA Is this discussion reaching a resolution?

**Comment by RussTreadon-NOAA** [comment]
@aerorahul:  `.nc` is the recommended suffix from the netcdf team.  Do EE2 standards offer any guidance?

**Reply by aerorahul** [reply]
No. .nc is fine.

**Comment by RussTreadon-NOAA** [comment]
OK.  Just need everyone to be aware of this convention and ensure that their netcdf files use the `.nc` suffix.  

Closing discussion.
